pkg/services/alerting/notifiers/googlechat.go

```diff
@@ -144,6 +144,7 @@ func (gcn *GoogleChatNotifier) Notify(evalContext *alerting.EvalContext) error {
 	}
 
 	// add a text paragraph widget for the fields
+	//nolint:prealloc // break block
```

Comment:
```suggestion
```
nit
I think `prealloc` [is not enabled](https://github.com/grafana/grafana/blob/6c6a970f56845cccbcea9e8d581f0187553899f2/.golangci.toml#L29-L62) in our configuration
so this comment is not necessary

Reply:
yes, i want to enable it, so i split big pr https://github.com/grafana/grafana/pull/59118